buildpipeline/Core-Setup-CrossBuild.json

```diff
@@ -49,7 +49,7 @@
       },
       "inputs": {
         "filename": "docker",
-        "arguments": "exec $(PB_DockerContainerName) git checkout $(SourceVersion)",
+        "arguments": "run --privileged -d -w $(PB_GitDirectory) --name $(PB_DockerContainerName) $(PB_DockerImageName) -v $(System.DefaultWorkingDirectory)$(PB_RepoName):$(PB_GitDirectory) sleep 7200",
```

Comment:
I think you need a '\' between these "$(System.DefaultWorkingDirectory)$(PB_RepoName)"